### ISSUE TITLE
search_explain is unusable with a 8.7 deployment

### ISSUE THREAD

**Issue by brnkes** [elastic/enterprise-search-js#40]
## 🐛 Bug Report

We can't use `app.search_explain` with our new deployment (8.7.0).

## To Reproduce

```js
const explanation = await esClient.app.search_explain({
    engine_name,
    body: {
      query: '',
      page: {
        size: 0
      }
    }
  });
```

## Expected behavior

I should receive a response in line with what's described in https://www.elastic.co/guide/en/app-search/8.7/search-explain.html#search-explain-api-examples

Instead, I'm getting :

```bash
ResponseError: {"errors":"Routing Error. The path you have requested is invalid."}
```

Our older deployments (of version 8.5.3) support both the `v0` and `v1` api routes so we're not running into the same issue on those.
Hitting the `/v1` route of our new deployment (with curl, postman, etc) works as expected.

Are there any plans to publish a new (8.7.x ?) package ? I can see that there is an update from a couple months ago resolving this issue (https://github.com/elastic/enterprise-search-js/commit/9bd804aaafb7dee91e85baf2a864afeb2fb912d1#diff-56e9f5dcb210718962163c215b0afd7cfd96c9838bc569a82489faf0fa30f604R904) but they're not present in the latest (8.6.0) `@elastic/enterprise-search` package published to npm.

## Your Environment

- *node version*: 18
- `@elastic/enterprise-search` *version*: 8.6.0
- *os*: Linux
- *Elastic Cloud deployment version*: 8.7

**Comment by jenseo** [comment]
+1 but for the searchEsSearch endpoint

**Comment by TattdCodeMonkey** [comment]
#52 will resolve this for [8.6 version](https://github.com/elastic/enterprise-search-js/pull/52/files#diff-56e9f5dcb210718962163c215b0afd7cfd96c9838bc569a82489faf0fa30f604R1014) of the library and this should be resolved in subsequent releases.

**Comment by TattdCodeMonkey** [comment]
This is fixed in the `8.6.1` release that is now available on npm
